client/components/line-chart/index.js

```diff
@@ -52,12 +53,14 @@ const dateFormatFunction = displayMonthTicksOnly => ( date, index, tickRefs ) =>
 };
 
 const dateToAbsoluteMonth = date => date.getYear() * 12 + date.getMonth();
+const NUM_SERIES = 3;
```

Comment:
Can we have a comment or a more descriptive name to that? I'm not sure what it does
It's the number of lines on the graph?